data/sql/updates/db_world/0000_00_00_00.sql

```diff
@@ -0,0 +1,325 @@
+
```

Comment:
the SQL file should be moved, please read: https://www.azerothcore.org/wiki/Dealing-with-SQL-files#how-to-create-an-sql-update-file

Reply:
done as requested